firmware/hw_layer/hardware.cpp

```diff
@@ -54,13 +54,13 @@
 #include "mc33816.h"
 #endif /* EFI_MC33816 */
 
-#if EFI_SPEED_DENSITY
```

Comment:
this was simply just the wrong flag - map avg doesn't care about whether SD is enabled.